cmd/boostd/import_data.go

```diff
@@ -80,6 +77,10 @@ var importDataCmd = &cli.Command{
 					return err
 				}
 
+				if deleteAfterImport {
+					return fmt.Errorf("legacy deal data cannot be automatically deleted after import (only new deals)")
```

Comment:
We could get to this point in the code if the user entered a proposal cid that doesn't exist. So maybe this error message should be
```
fmt.Errorf("cannot find boost deal with proposal cid %s and legacy deal data cannot be automatically deleted after import (only new deals)", proposalCid)
```